WordPress/WordPressTodayWidget/TodayViewController.swift

```diff
@@ -330,7 +391,17 @@ private extension TodayViewController {
 
     func expandedHeight() -> CGFloat {
         var height: CGFloat = 0
-        let dataRowHeight = tableView.rectForRow(at: IndexPath(row: 0, section: 0)).height
+        var dataRowHeight: CGFloat = 0
```

Comment:
Can this be made immutable since it doesn't change after setting the first time?

Reply:
Done!